pulsar/pulsar.go

```diff
@@ -17,33 +17,81 @@
 package pulsar
 
 import (
-	"bytes"
 	"crypto/ecdsa"
-	"crypto/rand"
 	"encoding/gob"
 	"errors"
-	"math/big"
 	"net"
 	"net/rpc"
+	"sync"
+	"time"
 
 	"github.com/insolar/insolar/configuration"
+	"github.com/insolar/insolar/core"
 	"github.com/insolar/insolar/log"
-	"golang.org/x/crypto/sha3"
+	"github.com/insolar/insolar/pulsar/storage"
 )
 
+type State int
+
+const (
+	WaitingForTheStart    State = 0
```

Comment:
use iota, see packet.go file.
You can use //go:generate stringer -type=packetType and log.Debug State switch